bool request
---
atf_msgs/TestStatus[] status
